tools/kompat/README.md

```diff
@@ -0,0 +1,9 @@
+# kompat
+
+Kompat is a simple CLI tool to interact with `compatibility.yaml` files which host your Kubernetes compatibility matrix.
+
+## Usage:
+
+```
+./kompat hack/compatibility-karpenter.yaml -n 5
```

Comment:
Maybe the `go install` command if someone wanted to run this outside the context of the Karpenter repo?